tailor-app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,11 @@
+<resources>
+    <string name="app_name">TailorMade</string>
+    <string name="your_design_label">Your Design</string>
+    <string name="add_label">Add</string>
+    <string name="dashboard_title_state">Hello, tailor!</string>
+    <string name="dashboard_description_state">Let’s start adding your designs to attract customer by tapping the button above !</string>
+    <string name="delete_menu_label">Delete</string>
+    <string name="select_all_label">Select All</string>
+    <string name="delete_design_alert_dialog_title">Are you sure?</string>
+    <string name="delete_design_alert_dialog_content">You’ll delete %d design(s)!</string>
```

Comment:
please check this for support prulars
https://developer.android.com/guide/topics/resources/string-resource#Plurals